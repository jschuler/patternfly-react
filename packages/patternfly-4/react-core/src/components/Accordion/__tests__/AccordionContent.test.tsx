// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { AccordionContent } from '../AccordionContent';

describe('AccordionContent test', () => {
  it('AccordionContent should match snapshot', () => {
    const view = shallow(
      <AccordionContent
        children={<>ReactNode</>}
				className={"''"}
				id={"''"}
				isHidden={false}
				isFixed={false}
				aria-label={"''"}
				component={undefined}
      />);
    expect(view).toMatchSnapshot();
  });
});
