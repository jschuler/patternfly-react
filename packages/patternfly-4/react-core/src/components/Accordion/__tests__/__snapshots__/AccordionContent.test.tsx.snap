// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccordionContent test AccordionContent should match snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
